samples/csharp/uwp/speechtotext-uwp/README.md

```diff
@@ -40,6 +40,7 @@ The app displays a graphical user interface (GUI).
 > **Note:**
 > if you are using your own `.wav` file as input source, make sure it is in the right format.
 > Currently, the only supported `.wav` format is **mono (single-channel), 16 kHz sample rate, 16 bits per sample**.
+> You may also use the [weather .wav file](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/csharp/dotnet-windows/speechtotext-wpf/speechtotext-wpf/audio) in the Speech samples.
```

Comment:
>https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/csharp/dotnet-windows/speechtotext-wpf/speechtotext-wpf/audio [](start = 43, length = 144)

could you update the destination location to
`[https://raw.githubusercontent.com/Azure-Samples/cognitive-services-speech-sdk/f9807b1079f3a85f07cbb6d762c6b5449d536027/samples/cpp/windows/console/samples/whatstheweatherlike.wav`](https://raw.githubusercontent.com/Azure-Samples/cognitive-services-speech-sdk/f9807b1079f3a85f07cbb6d762c6b5449d536027/samples/cpp/windows/console/samples/whatstheweatherlike.wav`)
?
Thanks, Mark

Reply:
Hi Mark, I tried this new link in a browser but it says "This audio not supported". Does it work in other browsers? I tried it in Edge.
@mahilleb-msft